go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/nokia/danm
+
+go 1.12
```

Comment:
can also upgrade to latest 1.13
main reason of not doing so was the outdated k8s code dependency

Reply:
Done